images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -104,7 +104,68 @@ _storage_overrides:
 meteringconfig_storage_hive_store_type: "{{ meteringconfig_default_values | combine(meteringconfig_spec_overrides) | json_query('storage.hive.type') }}"
 meteringconfig_storage_overrides: "{{ _storage_overrides[meteringconfig_storage_hive_store_type] | default({}, true) }}"
 
-meteringconfig_spec: "{{ meteringconfig_default_values | combine(meteringconfig_default_image_overrides, meteringconfig_storage_overrides, meteringconfig_spec_overrides, recursive=True) }}"
+#
+# Enable TLS by default
+#
+meteringconfig_reporting_operator_presto_server_ca_certificate: ""
+meteringconfig_reporting_operator_client_cert: ""
+meteringconfig_reporting_operator_presto_client_ca_certificate: ""
+meteringconfig_reporting_operator_client_key: ""
+
+meteringconfig_presto_server_secret_name: ""
+meteringconfig_presto_server_ca_certificate: ""
+meteringconfig_presto_server_certificate: ""
+meteringconfig_presto_server_key: ""
+meteringconfig_presto_tls_create_secret: false
+
+meteringconfig_presto_client_secret_name: ""
+meteringconfig_presto_client_ca_certificate: ""
+meteringconfig_presto_client_certificate: ""
+meteringconfig_presto_client_key: ""
+meteringconfig_presto_auth_create_secret: false
+
+_tls_overrides:
+  reporting-operator:
+    spec:
+      config:
+        prestoTLS:
+          enabled: true
+          createSecret: true
+          secretName: "reporting-operator-presto-server-tls"
+          caCertificate: "{{ meteringconfig_reporting_operator_presto_server_ca_certificate }}"
+
+        prestoAuth:
+          enabled: true
+          createSecret: true
+          secretName: "reporting-operator-presto-client-tls"
+          certificate: "{{ meteringconfig_reporting_operator_client_cert }}"
+          key: "{{ meteringconfig_reporting_operator_client_key }}"
+          caCertificate: "{{ meteringconfig_reporting_operator_presto_client_ca_certificate }}"
+  presto:
+    spec:
+      presto:
+        config:
+          tls:
+            certificate: "{{  meteringconfig_presto_server_certificate }}"
+            key: "{{ meteringconfig_presto_server_key }}"
+            caCertificate: "{{ meteringconfig_presto_server_ca_certificate }}"
+
+            enabled: true
+            createSecret: "{{ meteringconfig_presto_tls_create_secret }}"
+            secretName: "{{ meteringconfig_presto_server_secret_name }}"
+          auth:
+            certificate: "{{  meteringconfig_presto_client_certificate }}"
+            key: "{{ meteringconfig_presto_client_key }}"
+            caCertificate: "{{ meteringconfig_presto_client_ca_certificate }}"
+
+            enabled: true
+            createSecret: "{{ meteringconfig_presto_auth_create_secret }}"
```

Comment:
I'm wondering if we can just set this to `true` directly now. Any reason not to?